src/modules/twap/state/twapOrdersListAtom.ts

```diff
@@ -0,0 +1,35 @@
+import { atom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+
+import deepEqual from 'fast-deep-equal'
+
+import { tokensByAddressAtom } from 'modules/tokensList/state/tokensListAtom'
+import { walletInfoAtom } from 'modules/wallet/api/state'
+
+import { TwapOrderItem } from '../types'
+import { emulateTwapAsOrder } from '../utils/emulateTwapAsOrder'
+
+export type TwapOrdersList = { [key: string]: TwapOrderItem }
+
+export const twapOrdersListAtom = atomWithStorage<TwapOrdersList>('twap-orders-list:v2', {})
```

Comment:
Can we keep it `v1` until it's actually released to prod?

Reply:
Will reset it before release.